app/server/database/schema/readme.md

```diff
@@ -0,0 +1,43 @@
+# User Data Schema/Model
+
+The model/schema for the users has been designed inn the table below.  
+Please note: MongoDB automaticaly assigns an objectID to every document/user object entered so we can use that as the primary key for the document.
+
+| Field          | Data Type | Description                                        |
+| -------------- | --------- | -------------------------------------------------- |
+| userName       | String    | User Login Name, can serve as primary key          |
+| userPassword   | String    | User Password, may be encrypted for security       |
```

Comment:
I'm planning to use bcrypt for the encryption

Reply:
Okay, since you are planning to use bycrypt, a new issue doesn't need to be made, just remember when you tackle the issue of collecting details, to encrypt/hash them